libz-rs-sys/Cargo.toml

```diff
@@ -13,6 +13,11 @@ rust-version.workspace = true
 [lib]
 crate-type=["rlib", "cdylib"]
 
+[features]
+default = ["rust-allocator"] # when used as a rust crate, use the rust allocator
+c-allocator = [] # by default, use malloc/free for memory allocation
+rust-allocator = [] # by default, use the rust global alloctor for memory allocation
```

Comment:
I think this is the right default. When we build a dynamic library from this crate we should probably use the `c-allocator` to be consistent with stock zlib.

Reply:
Agreed